client/favorites.jsx

```diff
@@ -1,23 +1,65 @@
 import React from 'react';
 import NavBar from './navBar';
 import Loader from './loader';
+import fetch from 'node-fetch';
 
-export default class SearchResults extends React.Component {
+
+export default class Favorites extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      isLoading: false
+      favoritesData: '',
+      isLoading: true
     };
+    this.handleRemove = this.handleRemove.bind(this)
   }
 
+  handleRemove(e) {
+    e.preventDefault()
+    e.target.parentElement.className = 'fade1'
```

Comment:
In React we avoid direct DOM element manipulation. Instead, you should pass a different `className` prop to the element if you want to update its CSS classes.